### DISCUSSION TITLE
finishing self learning FRT

### DISCUSSION THREAD

**Discussion by Sudoo69** [github/india#336] [Students]
how are you guys going on about finishing your modules in self learning. I'm making like 5-6 hrs. a day vacant for this internship as I'm super busy but when I saw 52hrs more a single module I was like B R U H .

Anyway any advice will be helpful also I would like to know your schedule on how do you study and plan to finish it!

**Comment by TheSuranaverse** [comment]
@Sudoo69 
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together!
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Reply by Sudoo69** [reply]
Bro I'm already on your server stop the spam for the love of god
